src/app.tsx

```diff
@@ -19,10 +19,18 @@ if (process.env.NODE_ENV === 'production') {
 }
 
 export const App: React.FC = () => {
-    ReactGA.initialize('UA-51144540-39');
+    const options = {
+        gtagOptions: {anonymizeIp: true},
+        testMode: process.env.NODE_ENV === 'test'
```

Comment:
Oh woops - i merged this without a thorough review. The `process` global is only available in the node environment, not the browser... for simplicity I think we can just avoid setting this param altogether.

Reply:
Nevermind - I see we're already using `process.env.NODE_ENV` in other areas of this file. It's probably fine then I'll just leave it.